spec/scap_spec.rb

```diff
@@ -6,7 +6,7 @@
       allow(described_class).to receive(:openscap_available?).and_return(true)
       allow(described_class).to receive(:ssg_available?).and_return(true)
       allow(scap).to receive(:lockdown_profile)
-      expect { scap.lockdown("value1" => true) }.to raise_error
+      expect { scap.lockdown("value1" => true) }.to raise_error(String)
```

Comment:
What class would I use to catch https://github.com/ManageIQ/linux_admin/blob/master/lib/linux_admin/scap.rb#L26?
We could change that line to use a real error class, but this was just to get rid of the warnings while running the spec tests.